server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/SearchAction.java

```diff
@@ -62,7 +62,8 @@ public void define(WebService.NewController controller) {
       .setDescription(
         format("Language key. If provided, only profiles for the given language are returned. " +
           "It should not be used with '%s', '%s or '%s' at the same time.", PARAM_DEFAULTS, PARAM_PROJECT_KEY, PARAM_PROFILE_NAME))
-      .setPossibleValues(LanguageParamUtils.getLanguageKeys(languages));
+      .setPossibleValues(LanguageParamUtils.getLanguageKeys(languages))
+      .setDeprecatedSince("6.4");
 
     action.createParam(PARAM_PROJECT_KEY)
```

Comment:
As discussed I reverted my changes. :loop: Renaming the parameter will be done later.